models/course.js

```diff
@@ -0,0 +1,32 @@
+const { DataTypes } = require('sequelize')
+
+const sequelize = require('../lib/sequelize')
+const { User } = require('./user')
+
+const Course = sequelize.define('course', {
+  id: { type: DataTypes.INTEGER, allowNull: false },
+  subject_code: { type: DataTypes.STRING, allowNull: false },
+  number: { type: DataTypes.STRING, allowNull: false },
+  title: { type: DataTypes.STRING, allowNull: false },
+  term: { type: DataTypes.STRING, allowNull: false },
+  instructor_id: { type: DataTypes.INTEGER, allowNull: false }
```

Comment:
lets go over this together. we should set some to NOT NULL and include foreign keys